src/wp-includes/functions.wp-scripts.php

```diff
@@ -340,23 +346,29 @@ function wp_deregister_script( $handle ) {
  *                                    as a query string for cache busting purposes. If version is set to false, a version
  *                                    number is automatically added equal to current installed WordPress version.
  *                                    If set to null, no version is added.
+ * @param array            $args      {
+ *      Optional. An array of additional script loading strategies. Default empty array.
+ *
+ *      @type boolean   $in_footer    Optional. Default false.
+ *      @type string    $strategy     Optional. Values blocking|defer|async. Default 'blocking'.
+ * }
+ *
  * @param bool             $in_footer Optional. Whether to enqueue the script before `</body>` instead of in the `<head>`.
  *                                    Default 'false'.
```

Comment:
This still needs to be removed to have a correct doc block.

Reply:
Thanks @felixarntz , removed from here as well as the doc block for the `wp_enqueue_script` function.